src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -25,10 +27,10 @@ public void parse_validArgs_returnsFindCommand() {
         // no leading and trailing whitespaces
         FindCommand expectedFindCommand =
                 new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-        assertParseSuccess(parser, "Alice Bob", expectedFindCommand);
+        assertParseSuccess(parser, COMMAND_WORD + " n/Alice Bob", expectedFindCommand);
```

Comment:
Maybe in the future n/Alice Bob can be a static variable to make the test file neater. Similar issue found in line 33.

Reply:
Agree, fixed